src/partials/query.editor.html

```diff
@@ -1,7 +1,7 @@
 <query-editor-row query-ctrl="ctrl" can-collapse="false" has-text-edit-mode="true">
 <div ng-if="ctrl.target.rawQuery">
  <div class="gf-form">
-    <textarea rows="3" class="gf-form-input" ng-model="ctrl.target.query" spellcheck="false" placeholder="Druid SQL Query" ng-model-onblur ng-change="ctrl.refresh()"></textarea>
+    <textarea rows="20" class="gf-form-input" ng-model="ctrl.target.query" spellcheck="false" placeholder="Druid SQL Query" ng-model-onblur ng-change="ctrl.refresh()"></textarea>
```

Comment:
Why do you need this change?

Reply:
In general, when obtaining metrics directly using SQL, a complex query is required.
In this case, the SQL query will be longer, so the user will have to resize the input box each time.
To reduce this inconvenience, I expanded the default input box size.

Reply:
Got it